ablySpec/RealtimeClient.connection.swift

```diff
@@ -713,6 +713,56 @@ class RealtimeClientConnection: QuickSpec {
                 }
 
             }
+
+            // RTN12
+            context("close") {
+
+                // RTN12b
+                it("should receive CLOSED action within the default request timeout") {
```

Comment:
Ok, changed to `should transition to CLOSED action when the close process timeouts`.